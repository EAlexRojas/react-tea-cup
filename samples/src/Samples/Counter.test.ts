test("hey hoo", () => {
    expect(1).toBe(1)
});