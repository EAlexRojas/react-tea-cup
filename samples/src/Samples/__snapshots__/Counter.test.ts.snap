// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Counter view state snapshot 1`] = `
<div
  className="counter"
>
  <button
    onClick={[Function]}
  >
    -
  </button>
  <span>
    1313
  </span>
  <button
    onClick={[Function]}
  >
    +
  </button>
</div>
`;
